types/googlemaps/index.d.ts

```diff
@@ -3275,7 +3275,7 @@ declare namespace google.maps {
             reviews?: PlaceReview[];
             types?: string[];
             url?: string;
-            utc_offset?: number;
```

Comment:
`utc_offset` still exists and is described in [the documentation][utc_offset]. Better mark it as [`@deprecated`](https://jsdoc.app/tags-deprecated.html).

```js
/** @deprecated */
utc_offset?: number;
```

[utc_offset]: https://developers.google.com/maps/documentation/javascript/reference/places-service#PlaceResult.utc_offset